GitTfs.Vs2015/TfsHelper.Vs2015.cs

```diff
@@ -13,7 +13,7 @@ public TfsHelper(TextWriter stdout, TfsApiBridge bridge, IContainer container)
 
         protected override string GetDialogAssemblyPath()
         {
-            var tfsExtensionsFolder = TryGetUserRegString(@"Software\Microsoft\VisualStudio\14.0\ExtensionManager\EnabledExtensions", "Microsoft.VisualStudio.TeamFoundation.TeamExplorer.Extensions,14.0.23102");
+            var tfsExtensionsFolder = TryGetUserRegStringStartingWithName(@"Software\Microsoft\VisualStudio\14.0\ExtensionManager\EnabledExtensions", "Microsoft.VisualStudio.TeamFoundation.TeamExplorer.Extensions");
```

Comment:
I'm really not sure what is the better (for now and for the future). Because I think that this code will not move soon, I leave it as it is and have added some comments...